tests/integration_tests_control_files/reduced_order/CMakeLists.txt

```diff
@@ -46,12 +46,12 @@ add_test(
 #        WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 #)
 
-#configure_file(1d_burgers_rewienski_fd_sensitivity.prm 1d_burgers_rewienski_fd_sensitivity.prm COPYONLY)
-#add_test(
-#        NAME 1D_BURGERS_REWIENSKI_SENSITIVITY
-#        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_rewienski_fd_sensitivity.prm
-#        WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
-#)
+configure_file(1d_burgers_rewienski_fd_sensitivity.prm 1d_burgers_rewienski_fd_sensitivity.prm COPYONLY)
+add_test(
+        NAME 1D_BURGERS_REWIENSKI_SENSITIVITY
+        COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_1D -i ${CMAKE_CURRENT_BINARY_DIR}/1d_burgers_rewienski_fd_sensitivity.prm
+        WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
+)
```

Comment:
Likewise here, this test has been uncommented.

Reply:
Also commented out in recent commit.